allennlp/commands/train.py

```diff
@@ -564,6 +550,7 @@ def from_partial_objects(
             The directory where logs and model archives will be saved.
         local_rank: `int`
```

Comment:
`local_rank` is unused. Can it be removed from here?

Reply:
It's not actually unused, it gets passed along behind the scenes to the `Trainer`.